src/Utilities/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataAnalysis.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.FlowAnalysis;
+using Microsoft.CodeAnalysis.FlowAnalysis.DataFlow;
+using Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.PointsToAnalysis;
+
+namespace Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis
+{
+    using PointsToAnalysisResult = DataFlowAnalysisResult<PointsToBlockAnalysisResult, PointsToAbstractValue>;
+
+    internal partial class TaintedDataAnalysis : ForwardDataFlowAnalysis<TaintedDataAnalysisData, TaintedDataAnalysisContext, TaintedDataAnalysisResult, TaintedDataBlockAnalysisResult, TaintedDataAbstractValue>
+    {
+        private static readonly TaintedDataAnalysisDomain TaintedDataAnalysisDomainInstance = new TaintedDataAnalysisDomain(CoreTaintedDataAnalysisDataDomain.Instance);
+
+        private TaintedDataAnalysis(TaintedDataOperationVisitor operationVisitor)
+            : base(TaintedDataAnalysisDomainInstance, operationVisitor)
+        {
+        }
+
+        internal static TaintedDataAnalysisResult GetOrComputeResult(
+            ControlFlowGraph cfg,
+            Compilation compilation,
+            ISymbol containingMethod,
+            TaintedDataSymbolMap<SourceInfo> taintedSourceInfos,
+            TaintedDataSymbolMap<SanitizerInfo> taintedSanitizerInfos,
+            TaintedDataSymbolMap<SinkInfo> taintedSinkInfos)
+        {
+            WellKnownTypeProvider wellKnownTypeProvider = WellKnownTypeProvider.GetOrCreate(compilation);
+            PointsToAnalysisResult pointsToAnalysisResult = PointsToAnalysis.GetOrComputeResult(
+                cfg,
+                containingMethod,
+                wellKnownTypeProvider,
+                InterproceduralAnalysisKind.ContextSensitive,
+                true,
+                true);
+
+            TaintedDataAnalysisContext analysisContext = TaintedDataAnalysisContext.Create(
+                TaintedDataAbstractValueDomain.Default,
+                wellKnownTypeProvider,
+                cfg,
+                containingMethod,
+                InterproceduralAnalysisKind.ContextSensitive,
+                false /* pessimisticAnalysis */,
```

Comment:
named parameter? I am not sure if we moved our projects to latest C# language version, which allows you to specify named parameters in middle of positional arguments: [https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7-2#non-trailing-named-arguments](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7-2#non-trailing-named-arguments)